docs/book/src/plugins/declarative-v1.md

```diff
@@ -0,0 +1,60 @@
+# Declarative Plugin
+
+The declarative plugin allows you to create [controllers][controller-runtime] using the [kubebuilder-declarative-pattern][kubebuilder-declarative-pattern]. 
+By using the declarative plugin, you can make the required changes on top of what is scaffolded by default when you create a Go project with Kubebuilder and the Golang plugins (i.e. go/v2, go/v3). 
+
+<aside class="note">
+You can check samples using this plugin by looking at the `project-<go plugin version>-addon` samples under the [testdata][testdata] directory on the root directory of the Kubebuilder project.
+</aside> 
```

Comment:
```suggestion
<aside class="note">
<h1>Samples</h1>

You can check samples using this plugin by looking at the `project-<go plugin version>-addon` samples under the [testdata](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/) directory on the root directory of the Kubebuilder project.

</aside>
```

Seems that the note does not work with the link alias :-(

Reply:
That is strange, because it works in `plugins.md` - there an alias is also used inside the <aside> tag